static/app/views/monitors/components/monitorCheckIns.tsx

```diff
@@ -34,6 +31,22 @@ type State = {
   checkInList: CheckIn[];
 };
 
+const checkStatusToIndicatorStatus: Record<CheckInStatus, string> = {
+  [CheckInStatus.OK]: 'success',
+  [CheckInStatus.ERROR]: 'error',
+  [CheckInStatus.IN_PROGRESS]: 'muted',
+  [CheckInStatus.MISSED]: 'warning',
+  [CheckInStatus.TIMEOUT]: 'error',
+};
```

Comment:
Can you type it better than `string`?

I guess this needs updated first https://github.com/getsentry/sentry/blob/f23859ae33e227cbbd062d37924b41a1b7c9e358/static/app/components/statusIndicator.tsx#L6

Reply:
diff PR 👍

Reply:
will dooo